examples/fit_fake_data.py

```diff
@@ -30,7 +30,7 @@ def list_symbs(f):
 # }}}
 fl = figlist_var()
 # {{{creating a fake data recovery curve
-tau = nddata(r_[0:2:100j], "tau")
+tau = nddata(r_[0:2:256j], "tau")
```

Comment:
what's the point of this?

Reply:
testing when I feed eval a number that is different from that of the data